### PR TITLE
Fetch post users

### PR DESCRIPTION
**Pull request by shristabaruah**
- Added posts and users to the backend
- fetch posts  (Only members which the user follows) in home page
- Show all post in explore page 
- Show posts of user on  profile page
- show who to follow (users who are not followed)

### CODE REVIEW

File: src/Components/PostCard/PostCard.jsx [hunk 1]
```diff
@@ -15,14 +15,14 @@ import {
...
-  Tooltip,
 } from "@chakra-ui/react";
 import { BsThreeDots, BsFillBookmarkFill } from "react-icons/bs";
 import { MdModeEdit, MdDelete } from "react-icons/md";
 import { AiFillLike } from "react-icons/ai";
 import { Comment } from "../Comments/Comments";
 
-const PostCard = ({ img }) => {
+const PostCard = ({ img,post }) => {
+  console.log(post)
```
Comment (MarkVed17) [inline src/Components/PostCard/PostCard.jsx]: missed this console.

File: src/Components/Profile/Card-Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                variant="outline"
+                color="red.300"
+                bgColor="brand.400"
+                icon={<MdLogout />}
+                onClick={logoutHandler}
+              ></IconButton>
+            </Flex>
+            <Flex gap="4px">
+              <Text as="span"> 2 post</Text>
+              <Text as="span"> 3 Follwers </Text>
```
Comment (MarkVed17) [inline src/Components/Profile/Card-Profile.jsx:62]: Followers*

File: src/Components/Profile/Card-Profile.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+              <Text as="span"> 6 Follwing</Text>
```
Comment (MarkVed17) [inline src/Components/Profile/Card-Profile.jsx:63]: Following*

**Review by MarkVed17 — APPROVED** [review]
Good to proceed!

**Review by gadgetfather — APPROVED** [review]

**Pull request merged**